developing-with-nats/tutorials/jwt.md

```diff
@@ -0,0 +1,1540 @@
+Meaning, the Operator JWT is part of its server configuration and requires a restart or `nats-server --signal reload` once changed.
+It is also configured with a way to obtain account JWT in one of 3 ways (explained below).
+Clients provide a user JWT on connect. Again the server or Account JWT is not configured with them.
+They also possess the private NKEY corresponding the the JWT identity so they can proof their identity as described [above](#what-are-nkeys).
+
+Through NKEY signature verification involved JWT are validated and the hierarchy between them can be verified.
+
+##### Obtain an Account JWT
+
+To obtain an Account JWT, the nats-server is configured with one of three [resolver](https://docs.nats.io/nats-server/configuration/securing_nats/jwt/resolver) types.
+Which one to pick depends on your needs:
+* [mem-resolver](https://docs.nats.io/nats-server/configuration/securing_nats/jwt/resolver#memory): Very few or very static accounts
+  * You are comfortable changing the server config if operator or accounts change.
+  * You can generate user programmatically using NKEY and JWT library (more about that later).
+  * User do not need to be known by nats-server.
+* [url-resolver](https://docs.nats.io/nats-server/configuration/securing_nats/jwt/resolver#url-resolver): Very large volume of accounts
+  * Same as `mem-resolver`, except you do not have to modify server config if accounts are added/changed.
+  * Changes to the operator still require reloading (only few operations that require that).
+  * Will download Accounts from a web server.
+    * Allows for easy publication of account JWT programmatically generated using NKEY and JWT library (more about that later).
+    * The [`nats-account-server`](https://docs.nats.io/nats-tools/nas) is such a webserver. When set up correctly will inform `nats-server` of Account JWT changes.
+  * Depending on configuration, requires read and/or write access to persistent storage.
+* `nats-resolver`: Same as `url-resolver`, just uses nats instead of http
+  * No separate binary to run/config/monitor.
+  * Easier clustering when compared to `nats-account-server`. Will eventually converge on the union of all account JWT known to every participating `nats-server`.
+  * Requires persistent storage in form of a NON-NTFS directory for `nats-server` to exclusively write into.
+  * Optionally, directly supports Account JWT removal.
+  * Between this and `url-resolver`, the `nats-resolver` is the clear recommendation
+
+If your setup has few Accounts and User and/or you are comfortable reloading server configs when accounts/user change.
+Then safe yourself the complexity and do not use JWT. Regular config - possibly with NKEYs - will work just fine for you.
+
+##### JWT and Chain of Trust Verification
+
+Each JWT document has a subject it represents. This is the public identity NKEY represented by the JWT document.
+JWT documents contain an `issueAt` time when it was signed. 
```

Comment:
`issueAt` doesn't exist the jwt field is `iat`

Reply:
fixed